dataactvalidator/config/sqlrules/b27_award_financial_3.sql

```diff
@@ -0,0 +1,17 @@
+-- As of FY26, each row in file C must contain a PARK and must not contain a PAC/PAN.
+SELECT
+    row_number,
+    program_activity_reporting_key,
+    program_activity_code,
+    program_activity_name,
+    display_tas AS "uniqueid_TAS",
+    object_class AS "uniqueid_ObjectClass"
+FROM award_financial AS af
+    JOIN submission AS sub
+        ON sub.submission_id = af.submission_id
+WHERE af.submission_id = {0}
+    AND reporting_fiscal_year >= 2026
+    AND (COALESCE(program_activity_reporting_key, '') = ''
```

Comment:
Given rules B27.1/27.2 are already fatal for FY26 on if PARK is blank, we don't want to duplicate the fatal error here too; so I'm thinking, for both, you can drop the `COALESCE(program_activity_reporting_key, '') = ''` check too and just be an error if PAC/PAN is provided at all FY26 on.